src/cljs/org/broadinstitute/firecloud_ui/page/workspace/monitor/workflow_details.cljs

```diff
@@ -12,13 +13,13 @@
     [org.broadinstitute.firecloud-ui.utils :as utils]
     ))
 
+(def timingDiagramDrawn (atom false))
```

Comment:
I don't know anything about Clojure but this looks global-y to me. Is there a convention here?